msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -459,4 +470,27 @@ Specify your AAD App ID and Graph information to help users seamlessly sign into
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
 |`id`|String|36 characters|✔|AAD application id of the app. This id must be a GUID.|
-|`resource`|String|2048 characters|✔|Resource url of app for acquiring auth token for SSO.|
\ No newline at end of file
+|`resource`|String|2048 characters|✔|Resource url of app for acquiring auth token for SSO.|
+
+## configurableProperties
+
+**Optional** - array
+
+The `configurableProperties` block defines the app properties that Teams admin can customize.
```

Comment:
Can we link to concept doc page? For more information, see [customize apps in Microsoft Teams](/MicrosoftTeams/customize-apps).